clients/cli/src/prover.rs

```diff
@@ -356,17 +275,22 @@ async fn main() -> Result<(), Box<dyn std::error::Error>>{
             proof = prove_seq_step(Some(proof), &pp, &tr).expect("error proving step");
             steps_proven += 1;
             completed_fraction = steps_proven as f32 / steps_to_prove as f32;
-            let progress = ProverRequest {
-                contents: Some(prover_request::Contents::Progress(Progress {
-                    completed_fraction,
-                    steps_in_trace: total_steps as i32,
-                    steps_to_prove: steps_to_prove as i32,
-                    steps_proven,
-                })),
-            };
+  
+            
+
             let progress_duration = progress_time.elapsed();
-            let cycles_proven = steps_proven * 4;
             let proof_cycles_hertz = k as f64 * 1000.0 / progress_duration.as_millis() as f64;
+            
+            let progress = ClientProgramProofRequest {
```

Comment:
Can we throttle updates to once every 3 minutes? Can do in a separate PR

Reply:
Yes let’s do separate PR.